src/debug.c

```diff
@@ -1041,6 +1046,10 @@ void _serverAssertPrintClientInfo(const client *c) {
     serverLog(LL_WARNING, "client->conn = %s", connGetInfo(c->conn, conninfo, sizeof(conninfo)));
     serverLog(LL_WARNING, "client->argc = %d", c->argc);
     for (j = 0; j < c->argc; j++) {
+        if (j >= clientArgsToLog(c)) {
+            serverLog(LL_WARNING | LL_RAW, "client->argv[%d]: *redacted*\n", j);
```

Comment:
Instead of just redacted, can we include the length of the arg here? Might be useful for debugging.